charts/px-central/templates/px-backup/pxcentral-alertmanager.yaml

```diff
@@ -8,6 +9,11 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   configSecret: px-backup-alertmanager-custom-config
+  {{- if eq $azureProxyEnabled true }}
+  podMetadata:
+    annotations:
+      kubernetes.azure.com/no-http-proxy-vars: "true"
```

Comment:
nit: can this annotation be taken from a variable? Good to do that if possible

Reply:
This can be taken as an improvement, where we take any annotation from the User and apply it on all the specs